geetools/Image.py

```diff
@@ -1315,11 +1317,27 @@ def distance(self, other: ee.image) -> ee.Image:
 
         return ee.Image(distance)
 
-    def maskCover(self) -> ee.Image:
-        """return an image with the mask cover ratio as an image property.
+    def maskCoverRegion(
+        self,
+        region: ee.Geometry,
+        scale: Optional[int] = None,
```

Comment:
I understand what you say and I don't have an strong position on this.. the only reason why I'm not using the `**kwargs` pattern is to allow the user understand all parameters from reading the docstrings, instead of having to go to another code to see the documentation.

Reply:
but what if, GEE changes something ? it would force us to review continuously their code to make sure we match the complete length of their option. But I also see your point.... I actually complained about it to them for `ee.data.*` module.
as a descision needs to be taken I'm voting for **kwargs, do as your heart commands.